lib/maths/unittest/CLinearAlgebraTest.cc

```diff
@@ -1136,10 +1136,8 @@ BOOST_AUTO_TEST_CASE(testShims) {
     {
         BOOST_TEST_REQUIRE(TMatrix4(0.0) == maths::las::conformableZeroMatrix(vector1));
         BOOST_TEST_REQUIRE(TMatrix(4, 0.0) == maths::las::conformableZeroMatrix(vector2));
-        BOOST_TEST_REQUIRE(TDenseMatrix::Zero(4, 4) ==
-                           maths::las::conformableZeroMatrix(vector3));
-        BOOST_TEST_REQUIRE(TDenseMatrix::Zero(4, 4) ==
-                           maths::las::conformableZeroMatrix(vector4));
+        BOOST_REQUIRE(TDenseMatrix::Zero(4, 4) == maths::las::conformableZeroMatrix(vector3));
+        BOOST_REQUIRE(TDenseMatrix::Zero(4, 4) == maths::las::conformableZeroMatrix(vector4));
```

Comment:
What error did you get before making this change from `BOOST_TEST_REQUIRE` to `BOOST_REQUIRE`?  It suggests to me that one of these classes used to be printable but now isn't?

Reply:
I suspect that change was necessary when I initially tried to upgrade to the `master` version of Eigen. It doesn't seem to be needed now, removed.